gunicorn/config.py

```diff
@@ -820,12 +820,12 @@ class PreloadApp(Setting):
 class Sendfile(Setting):
     name = "sendfile"
     section = "Server Mechanics"
-    cli = ["--sendfile"]
+    cli = ["--no-sendfile"]
     validator = validate_bool
-    action = "store_true"
+    action = "store_false"
     default = True
     desc = """\
-        Enables or disables the use of ``sendfile()``.
+        Disables the use of ``sendfile()``.
 
         .. versionadded:: 19.2
```

Comment:
Will add a versionchanged directive. 

To address second suggestion, --sendfile never actually did _anything_ as far as I can tell. So unless someone can demonstrate otherwise, I don't see a benefit to keeping it around at all.